lang/c++/api/GenericDatum.hh

```diff
@@ -126,44 +128,44 @@ public:
     void selectBranch(size_t branch);
 
     /// Makes a new AVRO_NULL datum.
-    GenericDatum() : type_(AVRO_NULL), logicalType_(LogicalType::NONE) { }
+    GenericDatum() : type_(AVRO_NULL), logicalType_(LogicalType::NONE) {}
 
     /// Makes a new AVRO_BOOL datum whose value is of type bool.
```

Comment:
it might be worth a comment here, about why constructors are not marked explicit

Reply:
Did that